v0.7/speech_recognition/rnnt/accuracy_eval.py

```diff
@@ -31,13 +31,19 @@ def main():
     hypotheses = []
     references = []
     for result in results:
-        hypotheses.append(array.array('q', bytes.fromhex(result["data"])).tolist())
+        hypotheses.append(array.array('b', bytes.fromhex(result["data"])).tolist())
```

Comment:
@nvzhihanj We need to handle this using `output_dtype` flag, just like other benchmarks. default was `q`, which means int64. We should support int8, int16, int32, int64, and use int64 as default.

Reply:
Added